CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+CHANGES IN VERSION 1.11.0 (from 1.10.0)
+===================================
+
+Appium 1.11.0 is a minor release
+
+#### General
+
+#### iOS
```

Comment:
if there's nothing here, just not have the heading?

Reply:
Leaving that for now in case somebody thinks of something to add.... I'll remember to take it out though if we don't need it.

Reply:
https://github.com/appium/appium-xcuitest-driver/pull/855